src/components/Router/StepComponentMap.tsx

```diff
@@ -132,6 +133,7 @@ const buildCaptureStepComponents = (
   const faceStep = findStep('face')
   const documentStep = findStep('document')
   const dataStep = findStep('data')
+  const motionStep = findStep('motion')
```

Comment:
"motion" should be just a "variant" for the face step and not a separate step no?

adding to that, we don't want to support motion explicitly, but instead use or A/B approach to convert selfie/video to it so I guess it would be perhaps inside the `buildFaceComponents` that we would make that A/B logic and return either selfie/video or motion?

Reply:
Yeah, this is an outdated MR. I have changes to push that reflect how it's done in the other SDKs: the step will be private and will replace dynamically the `face` step.